packages/core/typings/js-joda.d.ts

```diff
@@ -1600,6 +1599,7 @@ export class YearMonth extends Temporal implements TemporalAdjuster {
     withMonth(month: number): YearMonth;
     withYear(year: number): YearMonth;
     year(): number;
+    toString(): string;
```

Comment:
Could you keep lexical order, move up after toJSON()

Reply:
done